CMakeLists.txt

```diff
@@ -268,16 +268,68 @@ IF(NOT DONTCREATEVFS)
 ENDIF(NOT DONTCREATEVFS)
 
 
-INSTALL(TARGETS astromenace DESTINATION ${CMAKE_INSTALL_PREFIX})
+# Desktop entry and icon installation for Linux (Freedesktop standard)
+IF(LINUX)
+    OPTION(INSTALL_DESKTOP_FILES "Install desktop entry and icon files" OFF)
+    IF(INSTALL_DESKTOP_FILES)
+        IF(NOT DEFINED DESKTOP_FILE_DIR)
+            SET(DESKTOP_FILE_DIR "share/applications")
+        ENDIF()
+        IF(NOT DEFINED ICON_DIR)
+            SET(ICON_DIR "share/icons/hicolor")
+        ENDIF()
+
+        INSTALL(FILES "${PROJECT_SOURCE_DIR}/share/astromenace.desktop" DESTINATION "${DESKTOP_FILE_DIR}")
+        INSTALL(FILES "${PROJECT_SOURCE_DIR}/share/astromenace_64.png"  DESTINATION "${ICON_DIR}/64x64/apps"   RENAME "astromenace.png")
+        INSTALL(FILES "${PROJECT_SOURCE_DIR}/share/astromenace_128.png" DESTINATION "${ICON_DIR}/128x128/apps" RENAME "astromenace.png")
+
+        MESSAGE(STATUS "Desktop files will be installed:")
+        MESSAGE(STATUS "  Desktop entry:  ${CMAKE_INSTALL_PREFIX}/${DESKTOP_FILE_DIR}/astromenace.desktop")
+        MESSAGE(STATUS "  Icon (64x64):   ${CMAKE_INSTALL_PREFIX}/${ICON_DIR}/64x64/apps/astromenace.png")
+        MESSAGE(STATUS "  Icon (128x128): ${CMAKE_INSTALL_PREFIX}/${ICON_DIR}/128x128/apps/astromenace.png")
+    ENDIF(INSTALL_DESKTOP_FILES)
+ENDIF(LINUX)
+
+
+IF(LINUX)
+  # Linux: default to FHS-style relative dirs
+  IF(NOT DEFINED CMAKE_INSTALL_BINDIR)
+    SET(CMAKE_INSTALL_BINDIR "bin" CACHE PATH "Directory for installing executables")
+  ENDIF()
+  IF(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+    SET(CMAKE_INSTALL_LIBDIR "lib" CACHE PATH "Directory for installing libraries")
+  ENDIF()
+  IF(NOT DEFINED GAME_DATA_DIR)
+    SET(GAME_DATA_DIR "share/astromenace" CACHE PATH "Directory for game data files (relative to CMAKE_INSTALL_PREFIX)")
+  ENDIF()

```

Comment:
As I mentioned early, you better make all changes separated by some IF/ENDIF only for package creation proccess. Changes you propose stil broke Linux user local build  (Build (macOS, Linux, BSD) section in README). If you move gamedata in this way you will be not able to start game just by execute binary.